CONTRIBUTING.md

```diff
@@ -144,7 +167,7 @@ Make their life easier.
 Archivematica uses the Python [PEP8](https://www.python.org/dev/peps/pep-0008/) community style guidelines.
 Newly-written code should conform to PEP-8 style.
 PEP8 is a daunting document, but there are very good linters available that you can run to check style in your code.
-The Archivematica development uses [Sublime Text](sublimetext.com/) with the [SublimeLinter-flake8](https://github.com/SublimeLinter/SublimeLinter-flake8) plugin, which automatically scans your code and displays any style notes inline.
+The Archivematica development team uses [Sublime Text](sublimetext.com/) with the [SublimeLinter-flake8](https://github.com/SublimeLinter/SublimeLinter-flake8) plugin, which automatically scans your code and displays any style notes inline.
```

Comment:
Sublime is non-free software, and is not used by all Artefactual employed developers, or all Archivematica committers, I don't think.  Can this be changed to a) strongly suggest the use of a linter with b) a preference for flake8 over other linters?